hack/vmimage.sh

```diff
@@ -3,12 +3,19 @@
 cleanup() {
   set +e
 
+  if [[ -e /var/run/secrets/kubernetes.io ]]; then
+    if [ $PHASE == "build_complete" ]; then
+      ARGS="-success"
+    fi
+    go run ./hack/ci-notify/main.go -job-name periodic-ci-azure-vmimage -comment "Phase: $PHASE" $ARGS
```

Comment:
Do we need to hard code `periodic-ci-azure-vmimage` here? Can we replace it with `$JOB_NAME`?